imagesets/imageset.sh

```diff
@@ -444,16 +452,18 @@ function google_update {
   google_find_old_objects
 
   TEMP_SETUP_SCRIPT="$(mktemp -t ${UNIQUE_NAME}.XXXXXXXXXX)"
+  # GCP doesn't allow periods in label values
+  TASKCLUSTER_VERSION="$(cat .taskcluster-version | tr '.' '_')"
```

Comment:
probably better to have separate variable with those substitutions for labels only, but keep the original one for the bootstrap substitutions
```suggestion
  TASKCLUSTER_VERSION="$(cat .taskcluster-version)"
  TASKCLUSTER_VERSION_LABEL="$(cat .taskcluster-version | tr '.' '_')"
```